source/multiprocessing/index.rst

```diff
@@ -11,82 +11,88 @@ Introduction
 
 Le module :py:mod:`multiprocessing` utilise les processus plutôt que les threads. Il nous permet de créer plusieurs processus séparés.
 
-    Bien que la plupart des CPUs modernes comportent plusieurs coeurs, le code que l’on écrit doit aussi être formatté adéquatement afin d’en tirer pleinement avantage. [#Ref]_
+Bien que la plupart des CPUs modernes comportent plusieurs coeurs, le code que l’on écrit doit aussi être formatté adéquatement afin d’en tirer pleinement avantage. [#Ref1]_
 
-Multiprocessing de python permet d'utiliser un ensemble de processus qui consumeront une liste de tâches contenues dans une :py:mod:`queue`.
+Multiprocessing de python permet d'utiliser un ensemble de processus qui consumeront une liste de tâches contenue dans une :py:mod:`queue`.
```

Comment:
Il y a plusieurs `queue` en Python.
- https://docs.python.org/3/library/queue.html
- https://docs.python.org/3/library/multiprocessing.html#pipes-and-queues
- https://docs.python.org/3/library/asyncio-queue.html

C'est la deuxième.